playwright/src/utilities/utilities.ts

```diff
@@ -68,25 +68,25 @@ class MobileNavigation {
   async openMobileMenu(menu: any) {
     switch (menu) {
       case 'toc':
-        await this.page.click('[aria-label="Click to open menu"]')
-        await this.page.click('[aria-label="Click to open the Table of Contents"]')
+        await this.page.click('[id*="mobile-menu-button"] [class*="styled__PlainButton"]')
```

Comment:
Just a thought, it might be good for us to add a `data-test-id` attribute to this element so that StyledComponent changes/upgrades don't break this.

Reply:
agree.